src/components/Board/Board.component.js

```diff
@@ -298,6 +378,75 @@ export class Board extends Component {
             'is-locked': this.props.isLocked
           })}
         >
+          {isLocked && (
+            <Joyride
+              callback={data => {
+                const { status } = data;
+                if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
+                  this.setState({ runLiveUnlock: false });
+                }
+              }}
+              steps={firststep}
+              continuous={true}
+              showSkipButton={true}
+              showProgress={true}
+              run={runLiveUnlock}
+              styles={{
```

Comment:
We want to be [D.R.Y.](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) as much as we can.
The `styles` object is duplicated (being used twice), should be just a variable, maybe:

```js
const joyRideStyles = {
  options: {
    ...
  }
};
```

Reply:
I made styles a variable named joyRideStyles.